### PR TITLE
Signup working and adding password strength feature. #1816

### LINKED ISSUE #1816: Signup Not working and missing password strength feature.
**I'm submitting a ...**
- [x] bug report
- [x] feature request

I am working on it.

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1816 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added missing meter 
- Fixed signup

### CODE REVIEW

File: frontend/app/templates/components/forms/signup-form.hbs [hunk 1]
```diff
@@ -23,17 +23,18 @@
                     <div class="ui left icon input">
                         <i class="lock icon"></i>
                         {{input type="password" id="pwd" value=password name="password" placeholder="Password"}}
-                        <div class="ui small basic icon buttons">
+                        <div classs="ui small basic icon buttons">
```
Comment (rajvaibhavdubey) [inline frontend/app/templates/components/forms/signup-form.hbs]: It should be class.

Comment (sk9331657) [reply frontend/app/templates/components/forms/signup-form.hbs]: Oops .

Comment (sk9331657) [reply frontend/app/templates/components/forms/signup-form.hbs]: Fixing this.

**Comment by sk9331657** [comment]
@gabru-md @yashLadha  Please approve this as new users are blocked currrentlt

**Pull request merged**